.github/workflows/build.yml

```diff
@@ -0,0 +1,28 @@
+# This workflow will build a Java project with Maven, and cache/restore any dependencies to improve the workflow execution time
+# For more information see: https://help.github.com/actions/language-and-framework-guides/building-and-testing-java-with-maven
+
+name: Java CI with Maven
+
+on:
+  pull_request:
+    branches: [ main ]
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v2
+    - name: Set up JDK 11
+      uses: actions/setup-java@v2
+      with:
+        java-version: '11'
+        distribution: 'adopt'
+        cache: maven
+    - name: Update
+      run: sudo apt-get update
+    - name: Install ffmpeg
+      run: sudo apt-get -y install ffmpeg
```

Comment:
I think these two steps should be combined similar to the example pattern in the docs: https://docs.github.com/en/actions/using-github-hosted-runners/customizing-github-hosted-runners#installing-software-on-ubuntu-runners

I think this is better since the "scope" of each command's work is the same; i.e., they're both working to get the latest package of ffmpeg

Reply:
Updated build logic.